src/content/blog/themes-catalog-updates.mdx

```diff
@@ -0,0 +1,76 @@
+import submitThemes from '/src/assets/blog/themes-catalog-updates/submit-themes.png'
+
+We [launched Astro Themes](https://astro.build/blog/themes-and-integrations/) earlier this year to highlight the growing list of Astro themes and starter templates being developed by our amazing community. Astro's [Themes Catalog](https://astro.build/themes/) makes it easy to browse a collection of themes for your next site and get started with a full-featured project from day one.
+
+**Today, we're excited to announce improvements to the themes catalog, support for paid themes, and an improved [theme submission process](https://astro.build/themes/submit).**
+
+<BlogContentImage
+    src={themesPreview}
+    alt="Screenshots of three Astro theme home pages"
+/>
+
+## Theme Detail Pages
+
+The initial release of our themes catalog focused on creating a launchpad for Astro themes, giving a quick overview of themes with a preview image and short description. As the list of themes grew that lead to an obvious question, what if you want to know more about a specific theme?
+
+<BlogContentImage
+    src={themesPreview}
+    alt="Screenshots of the new Astro themes catalog details page"
+/>
+
+Theme authors can now share more details about the theme they worked so hard to design, develop, and test. Multiple images can be included to show off different page templates, dark mode support, and included components. Badges add a quick reference to what tools and frameworks were used to build theme. Most importantly, a full description section means theme authors can share details that would never fit in a card layout.
+
+<Note title="Submit your theme">
+  Already have a theme in our catalog? Check out our new [submission form](https://astro.build/themes/submit/) to update your theme's details!
+</Note>
+
+## Premium Themes
+
+Building and maintaining a theme is a massive undertaking - from design and development to testing and user support the work really piles up fast.
+
+<BlogContentImage
+    src={paidThemes}
+    alt="Screenshots showing the details page for a paid theme on the new themes catalog"
+/>
+
+The catalog now supports premium themes, allowing theme authors to include a link to where the theme can be purchased. Whether a theme author uses [Gumroad](https://gumroad.com/) or sells on their own site, our catalog is making it easier to discover new themes **without** acting as a marketplace or taking a cut of sales.
+
+## New Submission Form
+
+Submitting your theme to the catalog just got easier.
+
+<BlogContentImage
+    src={submitThemes}
+    alt="An online form used to submit theme details with preview images, descriptions, and tags."
+/>
+
+Gone are the days of filling out GitHub issue templates - the new submission form that makes it simple to upload multiple preview images, select the tools used in your theme, and even includes a rich text editor for writing the full description of your theme.
```

Comment:
Nit: stray “that” and I think these could be two sentences for that extra punchiness ✨ 

```suggestion
Gone are the days of filling out GitHub issue templates. The new submission form makes it simple to upload multiple preview images, select the tools used in your theme, and even includes a rich text editor for writing the full description of your theme.
```